### ISSUE TITLE
Need some advice about training

### ISSUE THREAD

**Issue by jokerobbin** [CaptainEven/Vehicle-Car-detection-and-multilabel-classification#3]
Hi!
First of all, thanks for sharing these codes, especially the trained models. I've tested the detection part and the results are astonishingly good given the fact that yolov3-tiny has such a simple structure. So I'm really curious about your training approach. 
1.What dataset did you use for detection? I've tried it on coco and voc but the results were way worse than yours.
2.Is there any pre-trained weights for yolov3-tiny? or you just trained from scratch?
Hope you can help me!

**Comment by CaptainEven** [comment]
Hi, thanks for interest.
First, a big and good data-set is required, the company i work for has provided one(~2.5W) which i could not release out.  Thx for understanding.
Second, we take advantage of pre-trained weights of YOLO, namely fine-tune from imagenet data. It can be downloaded freely.
Third, we take an incremental tranning method : easy samples first, then hard samples for iterative training(use simple hand ratio of 6:4 for example).

**Comment by jokerobbin** [comment]
Thanks again!

**Comment by hustpenghuan** [comment]
@jokerobbin Hello! i'm also working on training this neural network.if it's possible, can you share your train code of the detection part with me?

**Comment by jokerobbin** [comment]
@hustpenghuan I trained the network on Darknet. You can check the code on Darknet's official site.
